### DISCUSSION TITLE
How can i get code hints when use pinia in webstore

### DISCUSSION THREAD

**Discussion by webszy** [vuejs/pinia#1156] [Help and Questions]
I‘m use pinia in my vue3 project(using Javascript), i choose webstorm to be my ide.i found when i use pinia,i can't get code hints.
for example:
first define a store
```
// store.js
import { defineStore } from 'pinia'
export const AppStore = defineStore('app', {
  state:()=>({
   list:[]
  }),
actions:{
  getData(){}
  }
})
```
then,i use this store on the other files

```
//xx.vue
<script setup>
import { AppStore  } from './store'
const app = AppStore()
app.get
</script>

```
I want webstorm can give me code hints when i typing app.get，but obviously not。how can fix it? I doesn't find any plugin or webpack plugin to do this

**Comment by STILLMOREzzz** [comment]
> I‘m use pinia in my vue3 project(using Javascript), i choose webstorm to be my ide.i found when i use pinia,i can't get code hints. for example: first define a store
> 
> ```
> // store.js
> import { defineStore } from 'pinia'
> export const AppStore = defineStore('app', {
>   state:()=>({
>    list:[]
>   }),
> actions:{
>   getData(){}
>   }
> })
> ```
> 
> then,i use this store on the other files
> 
> ```
> //xx.vue
> <script setup>
> import { AppStore  } from './store'
> const app = AppStore()
> app.get
> </script>
> ```
> 
> I want webstorm can give me code hints when i typing app.get，but obviously not。how can fix it? I doesn't find any plugin or webpack plugin to do this

If you use 2022.03.01 version of webstrom, you can solve this problem. I had the same problem earlier with 2022.1.3 version of webstrom
